### PR TITLE
add: import script for crowsupply data (first format of claim)

### PR DESCRIPTION
**Pull request by kenechukwu-orjiene**

### CODE REVIEW

File: import_claims/crowdsupply/import.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+def main():
+    settings = None
+
+    try:
+        with open("config.json") as f:
+            settings = json.load(f)
+    except Exception as e:
+        print_error_and_exit(str(e))
+
```
Comment (gvelez17) [inline import_claims/crowdsupply/import.py:67]: you shouldn't need to copy this

Comment (gvelez17) [reply import_claims/crowdsupply/import.py:67]: the whole idea of the config is that you can use common code?  the idea is to run `addclaims.py` with a parameter

you shouldn't copypasta same code, if you need some different can refactor but the idea is with the config file, to just have the one import script not a bunch of copies

**Review by gvelez17 — CHANGES_REQUESTED** [review]
please try to use existing add_pipe.py with config file, or if that is not enough then we should refactor.  lets not copy if we can avoid it

if you look at bcorp, noo, bbb note that these work without custom?  tho there is add-claims-with-amt.py

**Comment by kenechukwu-orjiene** [comment]
> please try to use existing add_pipe.py with config file, or if that is not enough then we should refactor. lets not copy if we can avoid it
> 
> if you look at bcorp, noo, bbb note that these work without custom? tho there is add-claims-with-amt.py

File: import_claims/crowdsupply/import.py (hunk 1 above)
Comment (kenechukwu-orjiene) [reply import_claims/crowdsupply/import.py:22]: Thank you. But I had some logics in some of the fields, should I tamper with the `addclaims.py`. 
These logics are just unique for import. What do you suggest please?

**Pull request closed without merging**